daikoku/app/storage/drivers/postgres/jooq/reactive/AbstractReactivePgAsyncClient.scala

```diff
@@ -0,0 +1,90 @@
+package storage.drivers.postgres.jooq.reactive
+
+import io.vertx.core.{AsyncResult, Handler}
+import io.vertx.sqlclient.{Row, RowSet, SqlClient}
+import org.jooq._
+import org.jooq.conf.ParamType
+import org.jooq.exception.TooManyRowsException
+import org.jooq.impl.DSL
+import play.api.Logger
+import storage.drivers.postgres.jooq.api.{PgAsyncClient, QueryResult}
+
+import java.util.concurrent.Executors
+import java.util.stream.StreamSupport
+import scala.concurrent.{ExecutionContext, Future, Promise}
+
+abstract class AbstractReactivePgAsyncClient[Client <: SqlClient](
+                                                                   val client: Client,
+                                                                   val configuration: Configuration)
+  extends PgAsyncClient {
+
+  implicit val ec: ExecutionContext = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(10))
```

Comment:
Maybe we need to make something more configurable here

Reply:
No JOOQ usage after PR #224 merging